main.html

```diff
@@ -24,24 +25,47 @@
                 <div class="header__burger">
                     <img src="burger.png" alt=""/>
                 </div>
+                <nav class="header-hidden__nav">
+                    <div class="header-hidden__list">
+                        <span class="header-hidden__text">DRIVER</span>
+                    </div>
+                    <div class="header-hidden__list">
+                        <span class=header-hidden__text>RIDER</span>
+                    </div>
+                    <div class="header-hidden__list">
+                        <span class=header-hidden__text>BUSINESS</span>
```

Comment:
I'd recommend using text-transform: uppercase in css instead of uppercase typing in html